packages/server/modules/core/services/objects.js

```diff
@@ -1,68 +1,15 @@
-'use strict'
 const { performance } = require('perf_hooks')
-const crypto = require('crypto')
 const { set, get, chunk } = require('lodash')
 
 const knex = require(`@/db/knex`)
 const { servicesLogger } = require('@/logging/logging')
-const { getMaximumObjectSizeMB } = require('@/modules/shared/helpers/envHelper')
-const { ObjectHandlingError } = require('@/modules/core/errors/object')
-const {
-  chunkInsertionObjectArray,
-  estimateStringMegabyteSize
-} = require('@/modules/core/utils/chunking')
+const { chunkInsertionObjectArray } = require('@/modules/core/utils/chunking')
+const { prepInsertionObject } = require('@/modules/core/services/objects/management')
 
 const Objects = () => knex('objects')
 const Closures = () => knex('object_children_closure')
 
 module.exports = {
-  /**
-   * @param {{streamId, object, logger?}} params
-   * @returns {Promise<string>}
-   */
-  async createObject({ streamId, object, logger = servicesLogger }) {
-    const insertionObject = prepInsertionObject(streamId, object)
-
-    const closures = []
-    const totalChildrenCountByDepth = {}
-    if (object.__closure !== null) {
-      for (const prop in object.__closure) {
-        closures.push({
-          streamId,
-          parent: insertionObject.id,
-          child: prop,
-          minDepth: object.__closure[prop]
-        })
-
-        if (totalChildrenCountByDepth[object.__closure[prop].toString()])
-          totalChildrenCountByDepth[object.__closure[prop].toString()]++
-        else totalChildrenCountByDepth[object.__closure[prop].toString()] = 1
-      }
-    }
-
-    delete insertionObject.__tree
```

Comment:
@didimitrie @gjedlicska removed these 2 lines cause there was no way for these props to ever be there - prepInsertionObject doesn't return them